### ISSUE TITLE
Algorithms don't work when sequences are different

### ISSUE THREAD

**Issue by vnnp** [life4/textdistance#57]
Algorithms damerau_levenshtein, levenshtein and hamming don't work when sequences are different.

If you run the code where the sequences are the same

```
v1 = [1,0,1,0,1,1,0,0,1]
v2 = [1,0,1,0,1,1,0,0,1]
print(textdistance.damerau_levenshtein(v1, v2))
print(textdistance.levenshtein(v1, v2))
print(textdistance.hamming(v1, v2))
```

it works

```
0
0
0
```

But if you run the code where the sequences are not the same

```
v1 = [1,0,1,0,1,1,0,0,1]
v2 = [1,1,1,1,1,1,1,1,1]
print(textdistance.damerau_levenshtein(v1, v2))
print(textdistance.levenshtein(v1, v2))
print(textdistance.hamming(v1, v2))
```

returns error

```
Traceback (most recent call last):
  File ".../textdist.py", line 15, in <module>
    print(textdistance.damerau_levenshtein(v1, v3))
  File ".../lib/python3.7/site-packages/textdistance/algorithms/edit_based.py", line 211, in __call__
    result = self.quick_answer(s1, s2)
  File ".../lib/python3.7/site-packages/textdistance/algorithms/base.py", line 96, in quick_answer
    answer = self.external_answer(*sequences)
  File ".../lib/python3.7/site-packages/textdistance/algorithms/base.py", line 73, in external_answer
    prepared_sequences = lib.prepare(*sequences)
  File ".../lib/python3.7/site-packages/textdistance/libraries.py", line 118, in prepare
    sequences = list(map(lambda x: u''.join(x), sequences))
  File ".../lib/python3.7/site-packages/textdistance/libraries.py", line 118, in <lambda>
    sequences = list(map(lambda x: u''.join(x), sequences))
TypeError: sequence item 0: expected str instance, int found
```

**Comment by orsinium** [comment]
It's a bug in adopting the sequence for an external lib. Please, use `external=False` (https://github.com/life4/textdistance#extra-libraries) as a temporary workaround.

**Comment by orsinium** [comment]
Fixed in #66
